assets/bundled/bbcode-parser.min.js

```diff
@@ -118,6 +118,66 @@
       );
     };
 
+    /**
+     * Add [block] tag
+     * @example [block=treasure]content[/block]
+     */
+    const block = (node) => {
+      const block = preprocessAttr(node.attrs)._default || "block";
+
+      const OPTIONS = [
+        "block",
+        "dice",
+        "dice10",
+        "setting",
+        "warning",
+        "storyteller",
+        "announcement",
+        "important",
+        "question",
+        "encounter",
+        "information",
+        "character",
+        "treasure",
+      ];
+
+      // Default to block option if user did not provide anything valid
+      const blockOption = OPTIONS.includes(block.toLowerCase()) ? block : "block";
```

Comment:
not sure, but I think this still allows `blockOption` to contain non-lowercase letters? so `DICE` will still be valid and the attributes will be `data-bb-block="DICE"`? Not sure if the mixin is case-sensitive.